### DISCUSSION TITLE
蓝奏云优享版没法预览文件也没法显示文件

### DISCUSSION THREAD

**Discussion by joyingwol** [AlistGo/alist#6441] [Q&A] [answered]
挂载后可以正常显示目录，但是点击文件的话会显示`failed link: failed get link: redirect failed, status: 200`。
![](https://s2.loli.net/2024/05/10/xVfXI79jzyhwY1O.png)
而且不论是本地代理还是302都是这样。

**Comment by Banrion-m** [comment]
一样的、请问解决了吗？

**Reply by ssyslk** [reply]
我也是这个问题

**Comment by huolanyun** [comment, marked as answer]
等待合并PR修复

**Reply by joyingwol** [reply]
> 等待合并PR修复

https://github.com/alist-org/alist/pull/6673

**Reply by joyingwol** [reply]
我引用一下对应的PR，感谢回复
